hnn_core/tests/test_drives.py

```diff
@@ -116,6 +116,30 @@ def test_drive_seeds(setup_net):
     assert ~np.any(np.allclose(trial1_spikes, trial2_spikes))
 
 
+def test_clear_drives(setup_net):
+    """Test clearing drives updates Network"""
+    net = setup_net
+    weights_ampa = {'L5_pyramidal': 0.3}
+    synaptic_delays = {'L5_pyramidal': 1.}
+
+    n_gids = net._n_gids
+    net.add_evoked_drive(
+        'prox', mu=40, sigma=8.33, numspikes=1,
+        weights_ampa=weights_ampa, location='proximal',
+        synaptic_delays=synaptic_delays, cell_specific=True)
+
+    assert len(net.external_drives) > 0
+    assert 'prox' in net.external_drives
+    assert 'prox' in net.gid_ranges
+    assert net._n_gids == n_gids + len(net.gid_ranges['L5_pyramidal'])
+
+    net.clear_drives()
+    assert len(net.external_drives) == 0
+    assert 'prox' not in net.external_drives
+    assert 'prox' not in net.gid_ranges
+    assert net._n_gids == n_gids
+
```

Comment:
Perhaps we can test adding a new drive after clearing drives to make sure the Network still works after clearing the drives?

Reply:
Mostly a check that the ranges and positions are populated as expected after resetting them.